app/controllers/api/v1/listen_signals_controller.rb

```diff
@@ -116,10 +134,38 @@ def create_response(message, type, response_group)
     Response.create_response(message, type, response_group)
   end
 
-  def create_grouped_response(default_response_msg, repeat_response_msg)
-    response_group = create_response_group
-    create_response(default_response_msg, Response::Type::DEFAULT, response_group)
-    create_response(repeat_response_msg, Response::Type::REPEAT, response_group)
+  def create_timed_response(message, expiration_date, response_group)
+    Response.create_timed_response(message, Response::Type::TIMED, expiration_date, response_group)
```

Comment:
If the method is called `.create_timed_response`, it seems superfluous to pass in the type of response. It seems like it should be hard coded within the method itself.